hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/TestKeyValueContainerMetadataInspector.java

```diff
@@ -44,9 +44,9 @@
     extends TestKeyValueContainerIntegrityChecks {
   private static final long CONTAINER_ID = 102;
 
-  public TestKeyValueContainerMetadataInspector(ContainerLayoutTestInfo
-      containerLayoutTestInfo) {
-    super(containerLayoutTestInfo);
+  public TestKeyValueContainerMetadataInspector(
+      ContainerTestVersionInfo versionInfo) {
+    super(versionInfo);
```

Comment:
Why we don't create RocksDB for SchemaV3 in this test class?

Reply:
Because this test class is a derived class from its base class `TestKeyValueContainerIntegrityChecks`, and the db instances are created there.